### ISSUE TITLE
Blynk cloud moving to new host blynk-cloud.com ?

### ISSUE THREAD

**Issue by Toshibass** [tzapu/node-red-contrib-blynk#27]
Hi Tzapu,  Regarding the post     http://community.blynk.cc/t/blynk-cloud-moving-to-new-host-blynk-cloud-com/4880    I am using blynk cloud with node-red-contrib-blynk and node-red-contrib-blynk-websockets, I did not download blynk library but I think its part of the download for node-red-contrib-blynk, could you offer me any advice on what I should do regarding this post advising that Blynk will move to new host ... blynk-cloud.com

**Comment by tzapu** [comment]
hi, i think you can just use blynk-cloud.com as a host and it should work

i will do an update in the next couple of weeks to the contrib-blynk to mention it s been replaced by the websockets variant, and will change the default host as well

thanks for letting me know

**Comment by Toshibass** [comment]
Yes for node-red-contrib-blynk-websockets changing ws://cloud.blynk.cc:8082/websocket ..to.. ws://blynk-cloud.com:8082/websocket works fine, I would move all my blynk nodes to websockets but for the fact that cloud ssl doesnt work at the moment.

**Comment by Toshibass** [comment]
Hi Tzapu, please do not forget about update for contrib-blynk default host, I am getting worried that blynk will close the blynk.cc server soon, resulting in 50% of my flows not working, as I said before I would move over completly to websocket variant but for the fact that I cannot currently use ssl with blynk cloud ... thanks

**Comment by Toshibass** [comment]
Hi Tzapu I had a go at this myself just modified the following line in..
 /home/pi/.node-red/node_modules/node-red-contrib-blynk/blynk.html

```
    label:  function() { 
        //var h = this.host?this.host:'cloud.blynk.cc';
        var h = this.host?this.host:'blynk-cloud.com';
        var p = this.port?this.port:'8441';
        var s = this.usessl?'tls':'tcp';
        return (s + '://' + h + ':' + p + '/' + this.key);
    }
```

Appears to work fine do you think I need to do anything else ?

**Comment by tzapu** [comment]
thanks for the pointer and snippet,
will try to do it today

is everything working now on `blynk-cloud.com` ?

cheers

**Comment by Toshibass** [comment]
Err define everything,  I mean every thing works that worked before on   cloud.blynk.cc  
![snip](https://cloud.githubusercontent.com/assets/6310911/14383019/17a21d46-fd8a-11e5-91e5-20f0dea61010.JPG)
by changing that line var h = this.host?this.host:'cloud.blynk.cc';     to    var h = this.host?this.host:'blynk-cloud.com'; 

I am getting Uncaught Exception maybe once in each 2 weeks but I made a flow on the pi that runs node-red-contrib-blynk-websockets with a blynk button that resets the node-red flows on the pi's running node-red-contrib-blynk so I can reset node-red from a button remotely, see attached.

I cannot get the bridge node to work no matter what i try see https://github.com/tzapu/node-red-contrib-blynk/issues/29  I made a flow that adds a keypad combination lock so I have a blynk project with 9 buttons that have to be pressed in the correct combination before any other projects buttons will work, works fine for all projects on that pi, but I need to extend that to all my other blynk projects on my other 2 pi's think if the bridge node could work I could just send a signal 0 or 1 or something to activate the buttons on those other projects as well.

I cannot figure out certs for local server so thats a dead end and in reality I want to use blynk cloud unless I can figure out a better way than vpn, it works generally but in a lot of hotels for instance they block vpn connections.

I cannot currently share a project from iphone to ipad but that's a bug in the blynk code they say should be fixed in next release

And of coarse ws://blynk-cloud.com:8082/websocket cannot use ssl yet which is a problem for me

Oh nice to get that of my chest hahahah

**Comment by tzapu** [comment]
man, you have some complicated stuff. nice :)

i ve got certs to work with startssl. i m sure the blynk guys will add certs to the live server in the next month (here s for hoping )

i ve updated the contrib to point to the right server and throw some deprecated messages.

i still get exceptions with this contrib , even after the last update
